tools/springboard-alerts/frameworks.json

```diff
@@ -91,23 +91,6 @@
             { "title": "“%@” Wants to Use “%@” to Sign In", "button": "Continue" }
         ]
     },
-    {
```

Comment:
I count 10 deletions here, but we have to delete 11 alerts. Am I missed something?